components/pkg-export-helm/src/main.rs

```diff
@@ -41,6 +44,18 @@ use hcore::PROGRAM_NAME;
 
 use chart::Chart;
 
+lazy_static! {
+    static ref CHART_VERSION_HELP: String = format!(
+        "Version of the chart to create (default: {})",
+        chartfile::DEFAULT_VERSION,
+    );
+
+    static ref OPERATOR_VERSION_HELP: String = format!(
+        "Version of the Habitat operator to set as dependency (default: {})",
```

Comment:
Can we add this now the k8s exporter CLI as well please :) (not part of this PR of course)